.github/workflows/warn-on-editing-requirements.yml

```diff
@@ -1,6 +1,10 @@
 name: Maintainer approval for requirement edits
 
 on:
+  pull_request_target:
+    paths:
+      - setup.py
+      - requirements/skinny-requirements.yaml
```

Comment:
```suggestion
```

These need to be removed. Once we mark this workflow as required (= triggered and succeeded), that would block PRs that don't update `setup.py`.

Reply:
More details here: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/collaborating-on-repositories-with-code-quality-features/troubleshooting-required-status-checks#example

Reply:
ah interesting, thanks for the reference!

Reply:
looks like it runs now. it feels a little weird to run it on every pull request, but i guess it will succeed due to the path check in `scripts`